src/vm-memory/src/lib.rs

```diff
@@ -46,7 +46,7 @@ fn build_guarded_region(
     flags: i32,
     track_dirty_pages: bool,
 ) -> Result<MmapRegion, MmapRegionError> {
-    let page_size = unsafe { libc::sysconf(libc::_SC_PAGESIZE) as usize };
+    let page_size = utils::get_page_size().expect("Cannot retrieve page size.");
```

Comment:
Should we return a `MmapRegionError` instead of panic ?

Reply:
I couldn't find any error variants in the vm-memory crate that would hint at this error. I see that it does not even check for this at all: https://github.com/rust-vmm/vm-memory/blob/acd281e9f76a45f331ac1dc3fee87b8ed1044f53/src/mmap_unix.rs#L199
I also think that it should never happen in practice, so I propose we leave it like this for now.